metrics.yaml

```diff
@@ -138,17 +120,25 @@ launch_event:
         type: string
 setting_event:
   current_browser:
-    type: labeled_string
+    type: event
     lifetime: ping
-    send_in_pings:
-      - settings
     description: The external browser has been changed
     bugs:
       - https://linktobugs.page
```

Comment:
I don't think we'll have a bug to list here since none of the work was tracked in a public issue tracker :/